harvard-university-of-worcester.csl

```diff
@@ -261,14 +261,14 @@
       </else>
     </choose>
   </macro>
-  <citation collapse="year" disambiguate-add-year-suffix="true" et-al-use-first="3" et-al-min="4">
+  <citation et-al-min="3" et-al-use-first="1" sort-separator="," disambiguate-add-year-suffix="true" collapse="year" cite-group-delimiter=" ">
```

Comment:
Fix is up now, with a [commit under processor tag 1.1.62](https://bitbucket.org/fbennett/citeproc-js/commits/003923f4b767e3148c9cc6cb38381bb686610f44). The updated processor has been pushed to the [propachi plugins](https://juris-m.github.io/downloads/#csl-stuff).